Makefile

```diff
@@ -382,7 +381,7 @@ ci:
 .PHONY: packages
 packages: planet-packages binary-packages teleport-package gravity-packages dns-packages\
 	rbac-app-package bandwagon-package tiller-package monitoring-package \
-	storage-package log-package k8s-packages telekube-packages selinux-policy-package
+	storage-package log-package k8s-packages telekube-packages
```

Comment:
I removed the policy package since this was never implemented. Will re-add once I've settled on the best way to deal with it.